ligo/README.md

```diff
@@ -1,14 +1,43 @@
-# BaseDAO LIGO implementation
+# BaseDAO LIGO
+
+__Note: This is still current in development.__
+
+This is the LIGO implementation of BaseDAO.
+
+The main difference between the two are:
+- `BaseDAO Lorentz`:
+    - uses compile-time configuration to configure various options of baseDAO, which means that each contract that uses `BaseDAO Lorentz` will have its own `michelson` contract.
+
+- `BaseDAO LIGO`:
+    - uses runtime configuration
+    - only one `michelson` contract is used, the user simply defines the configuration as part of originated storage
 
 ## Prerequisites
 
 You will need [ligo executable](https://ligolang.org/docs/intro/installation) installed.
 
 ## Build instructions
 
-Use Makefile to build a contract, for instance
-
+To build the LIGO contract, simply run:
 ```sh
 make all
 ```
 
+## Generating `Storage`
+TODO
+
+## Originating the contract
+To originate `baseDAO LIGO`, run:
+```
+TODO
+```
+
+## Testing
+
+`baseDAO LIGO` shares the tests with `baseDAO Lorentz` which are implemented in Haskell.
+
+To run the test:
```

Comment:
I think it's not necessary to repeat this here, let's just leave the line below pointing at the test's README for more info:
```suggestion
The tests for this contract are written in Haskell and you will need the `stack` tool to run them.

You can use the `Makefile` to do so as well:
```